src/state/store.ts

```diff
@@ -49,6 +49,7 @@ export const createRechartsStore = (
     middleware: getDefaultMiddleware =>
       getDefaultMiddleware({
         serializableCheck: false,
+        immutableCheck: process.env.NODE_ENV === 'test',
```

Comment:
Can we do !== prod instead? Pretty sure this doesn't get set to test when running things locally (besides tests)

Reply:
thats fine but I think you would need to negate that so that the check would be off for those values

Reply:
Certainly! My bad 😞 I will push a new commit shortly